src/manifold/src/manifold.cpp

```diff
@@ -767,4 +830,104 @@ Manifold Manifold::TrimByPlane(glm::vec3 normal, float originOffset) const {
 }
 
 ExecutionParams& ManifoldParams() { return params; }
+
+Manifold Manifold::Hull(const std::vector<float>& vertProps, const int numProp,
+                        const double precision) {
+  const int len = vertProps.size() / numProp;
+  // FIXME: Should there be a hull specific error?
+  if (len < 4) return Invalid();
```

Comment:
Why `std::vector<float>` instead of `std::vector<glm::vec3>`? `numProp` seems odd since this can only ever be used for 3D data, right?